Automattic/simplenote-android
# Update Passcode Layout

### Fix
Update the passcode layout with vector drawables and theme changes to match the app style.  See the screenshots and animations below for illustration.

![update_passcode](https://user-images.githubusercontent.com/3827611/59403724-07941480-8d61-11e9-8565-1ef90cebe5fb.png)

Before|After
-|-
![update_passcode_before](https://user-images.githubusercontent.com/3827611/59403568-5db48800-8d60-11e9-867d-b1a03888ed14.gif)|![update_passcode_after](https://user-images.githubusercontent.com/3827611/59403578-63aa6900-8d60-11e9-8ff6-97f0696686e8.gif)

### Test
1. Open navigation drawer.
2. Tap ***Settings*** item.
3. Tap ***Turn PIN lock on*** under ***PIN Lock*** section.
4. Enter PIN lock passcode.
5. Reenter PIN lock passcode.
6. Exit app.
7. Wait five seconds.
8. Open app.
9. Notice updated app logo, number buttons, and fingerprint image.
10. Long-press any number button.
11. Notice updated button background.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

#### Note
Since these changes require a `minSdkVersion` value of `21` or higher, the pull request will be in draft status until https://github.com/Automattic/simplenote-android/pull/689 is merged.
